### PR TITLE
fix(trade): display recipient on confirm screen

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

image

# To test

1. When recipient is set it should be displayed on confirm screen (swap/limit/twap)

### CODE REVIEW

File: apps/cowswap-frontend/src/modules/twap/const.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export const MINIMUM_PART_SELL_AMOUNT_FIAT: Record<SupportedChainId, CurrencyAmo
   [SupportedChainId.MAINNET]: CurrencyAmount.fromRawAmount(USDC[SupportedChainId.MAINNET], 1_000e6), // 1k
   [SupportedChainId.GNOSIS_CHAIN]: CurrencyAmount.fromRawAmount(USDC[SupportedChainId.GNOSIS_CHAIN], 5e6), // 5
   [SupportedChainId.ARBITRUM_ONE]: CurrencyAmount.fromRawAmount(USDC[SupportedChainId.ARBITRUM_ONE], 5e6), // 5
-  [SupportedChainId.SEPOLIA]: CurrencyAmount.fromRawAmount(USDC[SupportedChainId.SEPOLIA], 100e6), // 100
+  [SupportedChainId.SEPOLIA]: CurrencyAmount.fromRawAmount(USDC[SupportedChainId.SEPOLIA], 100e18), // 100
```
Comment (alfetopito) [inline apps/cowswap-frontend/src/modules/twap/const.ts:44]: Are you sure this is 18 decimals?

**Review by alfetopito — APPROVED** [review]

File: apps/cowswap-frontend/src/modules/twap/const.ts (hunk 1 above)
Comment (shoom3301) [reply apps/cowswap-frontend/src/modules/twap/const.ts:44]: image

Yes sir

File: apps/cowswap-frontend/src/modules/trade/containers/TradeBasicConfirmDetails/index.tsx [hunk 2]
```diff
@@ -143,6 +150,10 @@ export function TradeBasicConfirmDetails(props: Props) {
         />
       )}
 
+      {/*Recipient*/}
+
+      {/* Recipient */}
```
Comment (alfetopito) [inline apps/cowswap-frontend/src/modules/trade/containers/TradeBasicConfirmDetails/index.tsx:155]: Twice now?

```suggestion
      {/* Recipient */}
```

**Review by anxolin — COMMENTED** [review]
🙏

**Pull request merged**